examples/proc_FIR.py

```diff
@@ -23,21 +24,28 @@
         prscr.lookup_table,
     )
     for nodename in [
-        # "FIR_noPower",
+        "FIR_noPower",
         "FIR_34dBm",
     ]:
-        print(nodename)
+        fl.basename = nodename # this is a good example of how we can
+        #                        use basename to make it easy to deal
+        #                        with multiple datasets
         s = psd.find_file(
             thisfile,
             exp_type=exptype,
             expno=nodename,
             postproc=post_proc,
             lookup=prscr.lookup_table,
         )
+        # TODO ☐: if the units are not already set, that should be handled in
+        # the postproc.  (This was a problem with pyspecdata, and in any newer
+        # pulse sequence, the units should be set)
+        # NOTE: you can just delete "TODO" items as you resolve them -- no need
+        # to check them off.
         s.set_units("vd", "s")
-        clock_correction = True
         indirect = "vd"
         direct = "t2"
+        # TODO ☐: clock correction should be a new function in the library
```

Comment:
Please just move the code into a different module -- just like you recently did for `convert_to_power`